build-system-tests/templates/components/vue/App.vue

```diff
@@ -9,7 +9,6 @@
 
 <script lang="ts">
 import { defineComponent } from 'vue';
-// @ts-ignore
```

Comment:
Do we even need HelloWorld component?

Reply:
Right. We don't need the HelloWorld component at all. I didn't remove it because I didn't want to add an extra step which is not relevant to our test.